### PR TITLE
Italian language

### PR DESCRIPTION
**Pull request by DonPiotr**
I added it.ini.

### CODE REVIEW

File: res/lang/it.ini [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+err_pam_cred_err = impostazione delle credenziali fallita
+err_pam_cred_expired = credenziali scadute
+err_pam_cred_insufficient = credenziali insufficenti
+err_pam_cred_unavail = acquisizione delle credenziali fallita
+err_pam_maxtries = ragiunto massimo limite delle prove
+err_pam_perm_denied = permesso negato
+err_pam_session = errore della sessione
+err_pam_sys = errore del sistema
+err_pam_user_unknown = utente sconosciuto
+err_path = impostanzione del percorso predefinito fallita
```
Comment (kpetrilli) [inline res/lang/it.ini:27]: Typo impostanzione => impostazione

**Review by kpetrilli — COMMENTED** [review]
Found a typo at line 27

**Comment by kpetrilli** [comment]
Hi @DonPiotr I made a new Italian transation PR, since some of the strings you translated didn't make much sense to me. Feel free to check it out and comment it: #245

**Comment by AnErrupTion** [comment]
#245

**Pull request closed without merging**